client/src/components/DataEntry/BoardTable.tsx

```diff
@@ -71,32 +157,65 @@ const BoardTable: React.FC<IProps> = ({ boardName, ballots, url }: IProps) => {
         return ballot.status !== BallotStatus.NOT_AUDITED ? (
           <>
             {ballot.status === BallotStatus.AUDITED ? (
-              <StatusTag intent="success">Audited</StatusTag>
+              <TableColumn style={grayColor}>
+                <Icon icon="tick" /> Audited
+              </TableColumn>
             ) : (
-              <StatusTag intent="danger">Not Found</StatusTag>
+              <TableColumn style={grayColor}>
+                <Icon icon="tick" /> Not Found
+              </TableColumn>
             )}
-            <ReAuditBtn
-              to={`${url}/batch/${ballot.batch.id}/ballot/${ballot.position}`}
-              className="bp3-button bp3-small"
-            >
-              Re-audit
-            </ReAuditBtn>
           </>
         ) : (
-          <StatusTag intent="warning">Not Audited</StatusTag>
+          <WarningLabel>Not Audited</WarningLabel>
+        )
+      },
+    },
+    {
+      Header: 'Actions',
+      accessor: ballot => {
+        return ballot.status === BallotStatus.AUDITED ||
+          ballot.status === BallotStatus.NOT_FOUND ? (
+          <AuditBtn
+            to={`${url}/batch/${ballot.batch.id}/ballot/${ballot.position}`}
+            minimal
+            fill
+          >
+            Re-Audit
+          </AuditBtn>
+        ) : (
+          <AuditBtn
+            to={`${url}/batch/${ballot.batch.id}/ballot/${ballot.position}`}
+            minimal
+            fill
+          >
+            <strong>Audit Ballot</strong>
+          </AuditBtn>
```

Comment:
since these buttons have the same props except the inner text, could move the conditional to inside the text of the button